packages/packages.csv

```diff
@@ -58,9 +59,6 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
-Microsoft.AspNetCore.Internal.libuv-Linux,ship
-Microsoft.AspNetCore.Internal.libuv-Windows,noship
```

Comment:
Talked to @pranavkm about this and it's actually fine. These internal libuv packages are built and published via a different process, but this list here is only for things that are built as part of ASP.NET/EF, so it's not needed here.